src/ops/ops.go

```diff
@@ -195,6 +196,27 @@ func (o *ops) Reboot() error {
 	return nil
 }
 
+func (o *ops) SetBootOrder(device string) error {
+	_, err := o.ExecPrivilegeCommand(o.logWriter, "test", "-f", "/sys/firmware/efi")
+	if err != nil {
+		o.log.Info("efi not supported")
+		return nil
+	}
+
+	o.log.Info("Setting efibootmgr to boot from disk")
+	_, err = o.ExecPrivilegeCommand(o.logWriter, "mount", device+"2", "/mnt")
+	if err != nil {
+		o.log.Errorf("Failed to mount device %s, err: %s", device+"2", err)
+		return err
+	}
+	_, err = o.ExecPrivilegeCommand(o.logWriter, "efibootmgr", "-d", device, "-p", "2", "-c", "-L", "RHCOS", "-l", "\\EFI\\BOOT\\BOOTX64.EFI")
```

Comment:
Shouldn't it be followed by a efibootmgr --bootorder?

Reply:
Good point. It seems to set the new one created as the first in the order but explicit is always better.

Reply:
Looked into it and we don't need an explicit boot order using -n here. -c will add it to the top of the order.
`-c | --create         create new variable bootnum and add to bootorder`